src/helpers/github.js

```diff
@@ -1,11 +1,45 @@
 const { generateGitHubIssueBody } = require("./utils");
 
+/**
+ * Get User in Organization
+ */
+
+const getGithubUserData = async (orgName, user) =>
+{
+  try
+  {
+    const apiUrl = `https://api.github.com/orgs/${orgName}/memberships/${user}`;
```

Comment:
Could be very interesting to have some type of simple OAuth application that will post the issue on behalf of the user that sent the original message. This will allow them to be eligible for the comment incentives for creating a new issue.

Reply:
> Could be very interesting to have some type of simple OAuth application that will post the issue on behalf of the user that sent the original message. This will allow them to be eligible for the comment incentives for creating a new issue.

I actually thought about this lol, it'd be really cool.. so they just visit the bot and send `/setup` or something and it generates 0auth page to link the user

Reply:
I think it might make more sense on the Telegram side in a private message so that nobody front runs clicking their generated link for example.

Reply:
> I think it might make more sense on the Telegram side in a private message so that nobody front runs clicking their generated link for example.

Yea, that is what i meant.. they message the bot directly privately